src/Centreon/Infrastructure/Monitoring/MonitoringRepositoryRDB.php

```diff
@@ -910,7 +910,7 @@ public function findServicesByIdsForNonAdminUser(array $serviceIds): array
                 $services[] = $service;
             }
         } else {
-            throw new \Exception(_('Bad SQL request'));
+            throw new \Exception(_('Failed to find services'));
```

Comment:
No need to throw an exception. If an exception is thrown by PDO it is up to the service to handle the case.

Reply:
done :heavy_check_mark: